apps/io-wallet-user-func/src/app/main.ts

```diff
@@ -255,3 +260,12 @@ app.storageQueue("callPidIssuerRevokeApi", {
   queueName:
     config.azure.storage.walletInstances.queues.pidIssuerRevokeApi.name,
 });
+
+app.storageQueue("sendEmailOnWalletInstanceCreationQueue", {
+  connection: "StorageConnectionString",
+  handler: SendEmailOnWalletInstanceCreationFunction({
```

Comment:
The handler function's name should match the one in the function app (expect for `Function`)
btw I would prefer `sendEmailOnWalletInstanceCreation`